src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -507,6 +597,19 @@ private static CodeWriter.CodeWriterScope WriteClientMethodDeclaration(CodeWrite
             return scope;
         }
 
+        private static CodeWriter.CodeWriterScope WriteClientMethodDeclaration(CodeWriter writer, string clientTypeName, LowLevelClientMethod clientMethod, bool async, BuildContext<LowLevelOutputLibrary> context)
+        {
+            var methodSignature = clientMethod.Signature.WithAsync(async);
+
+            WriteMethodDocumentation(writer, methodSignature, clientMethod);
+            var exampleComposer = new LowLevelExampleComposer(clientTypeName, context);
```

Comment:
It looks like this instance can be shared between all methods.

Reply:
Yes. I also thought of that. I planned to do a refactoring on `LowLevelClientWriter` to remove some statis modifier so that this instance can be shared. Let me try to do the minimal here to make it shared.

Reply:
I change to create the example composer in higher level of the callstack, so that it can be shared in each method generation.